mne/io/nirx/nirx.py

```diff
@@ -133,11 +135,15 @@ def __init__(self, fname, preload=False, verbose=None):
                                 hdr['DataStructure']['S-D-Key'])], int)
 
         # Determine if short channels are present and on which detectors
-        has_short = np.array(hdr['ImagingParameters']['ShortBundles'], int)
-        short_det = [int(s) for s in
-                     re.findall(r'(\d+)',
-                     hdr['ImagingParameters']['ShortDetIndex'])]
-        short_det = np.array(short_det, int)
+        if any(key == 'shortbundles' for key in hdr['ImagingParameters']):
```

Comment:
Can this just be `if 'shortbundles' in hdr['ImagingParameters']`? If `hdr` is dict-like then it should work